server/market/types/types.go

```diff
@@ -0,0 +1,151 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/decred/dcrdex/server/asset"
+)
+
+// MarketInfo specified a market that the Archiver must support.
+type MarketInfo struct {
+	Name    string
+	Base    uint32
+	Quote   uint32
+	LotSize uint64
+}
+
+func marketName(base, quote string) string {
+	return base + "_" + quote
```

Comment:
This is the naming scheme in use for the DB schemas, but I put it here since since it's a reasonable string representation for a market.  Perhaps we can duplicate this in the db/driver/pg package.